tests/QueryTest.php

```diff
@@ -239,6 +239,17 @@ public function testOrder(): void
         $this->assertEquals(35, $user->age);
     }
 
+    public function testStringableOrder(): void
+    {
+        $age = new stringableObject('age');
```

Comment:
```suggestion
        $age = str('age');
```

See @ https://laravel.com/docs/9.x/helpers#method-str